ui/src/views/Clients/ViewClient/index.jsx

```diff
@@ -99,6 +100,15 @@ export default class ViewClient extends Component {
     },
   };
 
+  getClientFormKey = memoize(initialClient => JSON.stringify(initialClient), {
+    serializer: (initialClient, query) => {
+      // Use expires as cache key
+      const serializeKey = query ? query.expires : 'no-expires';
```

Comment:
Is there a way to use the initial client as the serialize key?

Reply:
Maybe use `initialClient` but omit `expires`? Since `expires` will be different every time component rerender

Reply:
Yep, that's what I had in mind.

Reply:
OK, I see! Let me update it

Reply:
Updated!